game-core/src/main/java/games/strategy/debug/TextUtils.java

```diff
@@ -0,0 +1,14 @@
+package games.strategy.debug;
+
+import com.google.common.html.HtmlEscapers;
+
+final class TextUtils {
+  private TextUtils() {}
+
+  static String textToHtml(final String text) {
```

Comment:
Unfortunately, I couldn't host this method in `ErrorMessage` because the unit tests would raise a `HeadlessException` when run on Travis due to `ErrorMessage` executing Swing code when the class (enum) is initialized.